approve.php

```diff
@@ -1,8 +1,36 @@
 <?php
-// connect database
+// Start session
+session_start();
+
+// Connect to the database
```

Comment:
I think there might be a built in function that does the database connection

Reply:
Sounds good I'll use that to clean the code up.